### ISSUE TITLE
Spyder Crashes When Opening MATLAB Layout

### ISSUE THREAD

**Issue by robotdna** [spyder-ide/spyder#8005]
### What steps will reproduce the problem?

1. Open Spyder
2. Select: View > Window Layouts > Matlab Layout
3. Spyder crashes

### Traceback
```python-traceback
TypeError: _start_server() missing 1 required positional argument: 'port'
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.7.0
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Linux 4.15.0-34-generic

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.7.9 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.12.1 (OK)
psutil >=0.3      :  5.4.7 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.15.1 (OK)
sympy >=0.7.3     :  1.2 (OK)
cython >=0.21     :  0.28.5 (OK)
qtconsole >=4.2.0 :  4.4.1 (OK)
IPython >=4.0     :  6.5.0 (OK)
matplotlib >=2.0.0:  2.2.3 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

**Comment by ccordoba12** [comment]
This is not a problem with the Matlab layout but with Python 3.7. It's already fixed and it'll be part of our next release (3.3.2). In the meantime, please use Python 3.6

**Comment by robotdna** [comment]
Excellent, thanks!
